tools/ovc/openvino/tools/ovc/convert.py

```diff
@@ -70,9 +70,9 @@ def convert_model(
         :param output:
             The name of the output operation of the model or list of names. For TensorFlow*,
             do not add :0 to this name. The order of outputs in converted model is the
-            same as order of specified operation names. For PaddlePaddle model represented
-            as a Python object, you can specify outputs as a PaddlePaddle Python objects or
-            a list of such objects.
+            same as order of specified operation names. Example: output="out_1", or output=["out_1", "out_2"]).
```

Comment:
you have extra parenthesis here `]` or `)`

Reply:
How should we specify outputs with names containing commas for `ovc` tool?

Reply:
>you have extra parenthesis here ] or )

fixed.

>How should we specify outputs with names containing commas for ovc tool?

Unfortunately, we cannot specify outputs with commas in ovc tool. In that case user must use python api. @slyalin insisted on that scenario instead of adding additional separators in cli tool (which would be complicated).